gate/gate-web/src/test/resources/gate-test.yml

```diff
@@ -1,4 +1,7 @@
 spring:
+  mvc:
```

Comment:
Why do we need this here if BuildControllerSpec brings in Main and Main specifies this?

Reply:
My goal would be to minimize the diffs, and to make sure we understand whatever changes we're making.